komodo/cli.py

```diff
@@ -113,17 +113,6 @@ def _main(args):
 
     create_enable_scripts(komodo_prefix=tmp_prefix, komodo_release=args.release)
 
-    if args.locations_config is not None:
-        with open(args.locations_config, mode="r", encoding="utf-8") as defs, open(
-            Path(args.release) / "local", mode="w", encoding="utf-8"
-        ) as local_activator, open(
```

Comment:
Just out of curiosity what's the difference between `locations` and `local_activators`?

Reply:
I am not sure. I removed it because `args.locations_config is not None:` will never be true without locations.

Reply:
the "local activator"s are the shell scripts that are run by the enable script, so they should be removed yes.